packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -344,7 +344,7 @@ export default defineComponent({
 			return Math.min(this.runOutputIndex, this.maxOutputRun);
 		},
 		maxInputRun(): number {
```

Comment:
LGTM. Can you please add tests to address this issue?

Reply:
Done